yarn/src/main/scala/org/apache/spark/deploy/yarn/ApplicationMaster.scala

```diff
@@ -503,8 +513,8 @@ private[spark] class ApplicationMaster(
     val userThread = new Thread {
       override def run() {
         try {
-          val mainArgs = new Array[String](args.userArgs.size)
-          args.userArgs.copyToArray(mainArgs, 0, args.userArgs.size)
+          val mainArgs = new Array[String](userArgs.size)
+          userArgs.copyToArray(mainArgs, 0, userArgs.size)
```

Comment:
This comes from all the way back when there was `yarn/alpha` and `yarn/stable`, but yeah, it doesn't look necessary.